### PR TITLE
Fix hyperlinks and add gitbub workflow

### PR DESCRIPTION
**Pull request by purna135**

### CODE REVIEW

**Comment by purna135** [comment]
I have a question, why do we need this [workflow](https://github.com/pymc-devs/pymcon_2022_website/tree/main/.workflows) file here ?

**Comment by purna135** [comment]
Should we delete the `gh-pages` branch that already exists?
because the GitHub workflow will create a branch with the same name once we merge this PR

**Comment by canyon289** [comment]
> I have a question, why do we need this [workflow](https://github.com/pymc-devs/pymcon_2022_website/tree/main/.workflows) file here ?

it seems like we dont anymore. Lets delete it.

> Should we delete the `gh-pages` branch that already exists?
> because the GitHub workflow will create a branch with the same name once we merge this PR
Done

File: layouts/_default/contact.html [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {{ define "customCSS" }}
-    {{ $baseURL := .Site.BaseURL }}
-    <link rel="stylesheet" href="{{ $baseURL }}css/contact.css">
+    <link rel="stylesheet" href="/css/contact.css">
     <style>
         body {
-            background-image: url("{{ $baseURL }}images/bg-texture.png");
```
Comment (canyon289) [inline layouts/_default/contact.html:6]: Oh man I didnt realize this had to be done manually. Sorry if this took a lot of time

**Pull request merged**